lib/src/component_declaration/component_base.dart

```diff
@@ -514,25 +494,7 @@ abstract class UiProps extends MapBase
     }
   }
 
-  // Manually implement members from `MapViewMixin`,
```

Comment:
This branch looks so much better than what we currently have! 

There are a few more things that could be cleaned up, though, and since they're hard to explain I've just made a PR into this branch: https://github.com/Workiva/over_react/pull/158.